workshop/exercise-3/README.md

```diff
@@ -1,57 +1,82 @@
 ## Deploy our First Application to Knative using the Knative Client (kn)
-The knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
+The Knative client, `kn`, aims to make interacting with Knative a seamless experience for developers. In this section we'll try deploying an application using `kn`. The application for this lab is a simple Node.js with Express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
 
-# Our First Knative Service
-Knative Serving supports deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
+### Our First Knative Service
+Knative Serving enables rapid deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-The Knative Service object automatically manages the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new revision for each update of the service.
+The Knative Service object automatically manages the whole lifecycle for your workload. A Service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new Revision for each update of the service.
 
-![](https://github.com/knative/serving/raw/master/docs/spec/images/object_model.png)
+![](https://github.com/knative/serving/raw/master/docs/spec/images/fibknativev1.png)
 
-# Deploy our Application to Knative using kn
+### Deploy our Application to Knative using kn
 We've already created an image on dockerhub that contains the first version of our Fibonacci application. If we call the `/` endpoint, and pass in a `number` parameter, we should get the first `n` numbers of the Fibonacci sequence.
 
-1. Deploy the application:
+1. Deploy the application. We will create a Knative Service named fib-knative which will run our fib-knative image on dockerhub.
 
     ```
-    kn service create --image docker.io/ibmcom/fib-knative fib-knative
+    kn service create fib-knative --image docker.io/ibmcom/fib-knative
```

Comment:
No biggie, but I'm curious... why do you bother to put the `docker.io/` part of the image name?  Just to be extra explicit?

Reply:
yeah, i think it's helpful to be explicit about where that image is.

Reply:
I generally don't like there to be "magic" when i'm going through a tutorial or learning something the first time.